embedded.go

```diff
@@ -1,4 +1,4 @@
-package tracee
+package ebpf
 
 import _ "embed"
```

Comment:
Why do we even need this file?
It only contains embed for rego helpers, which are not used in the ebpf (tracee) packge, are they?

Reply:
we needed a file that would only embed things if "ebpf" build flag is given. The other will always embed things. This has been a result of the merge of tracee-ebpf and tracee-rules. Please make sure that files are correctly embedded if this is changed (BTFHUB=1 and check if btfhub is being embedded).

Reply:
> we needed a file that would only embed things if "ebpf" build flag is given. The other will always embed things. This has been a result of the merge of tracee-ebpf and tracee-rules. Please make sure that files are correctly embedded if this is changed (BTFHUB=1 and check if btfhub is being embedded).

But what do you embed here? currently nothing (maybe it was used when we have to embed the C sources before we removed it from main.go? If so, this file can be removed as well)

Reply:
Yea i'm also a bit confused about the use of `embedded.go` in the root directory. I changed it to 'package ebpf' with search/replace, but now that I think about it, it's not imported anywhere, is it? It embeds the signautres/rego/helpers.rego when built but I'm not sure it's actually being built.

Reply:
Related commits:

commit 82a1289c
Author: Jose Donizetti <jdbjunior@gmail.com>
Date:   Mon Aug 9 17:32:35 2021

    tracee-ebpf: add support to custom rego helpers

and

commit c22f59cd
Author: Simarpreet Singh <simar@linux.com>
Date:   Tue Mar 9 19:14:40 2021

    feat: Use //go:embed to bundle artifacts (#596)

    Signed-off-by: Simarpreet Singh <simar@linux.com>

Reply:
Ok that makes sense. After testing it I reverted back to keeping the embedded go files as the 'github.com/aquasecurity/tracee' package

Reply:
I don't think I follow...
Why do we need this file?